examples/reference/layouts/Fade.ipynb

```diff
@@ -0,0 +1,122 @@
+    "\n",
+    "This can also support more than two objects and will gradually fade between object 0 to object 1 to object 2.\n",
+    "\n",
+    "#### Parameters:\n",
+    "\n",
+    "* **``objects``** (list): The before and after components to lay out.\n",
+    "* **``value``** (int): The percentage value of the slider. Default is 0.\n",
+    "\n",
+    "Styling-related parameters:\n",
+    "\n",
+    "* **``slider_width``** (int): The width of the slider in pixels. Default is 12.\n",
+    "* **``slider_color``** (str): The color of the slider. Default is 'black'.\n",
+    "\n",
+    "For further layout and styling-related parameters see the [customization user guide](../../user_guide/Customization.ipynb).\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5613561d-d483-45df-8cbb-366abbb96e79",
+   "metadata": {},
+   "source": [
+    "The `Fade` layout accepts any objects, which must have identical sizing options to work as intended.\n",
+    "\n",
+    "Here we compare two images of mean surface temperatures in 1945-1949 and temperatures between 2015-2019:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "07cdc635-9642-4335-9862-d8195dadade7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gis_1945 = 'https://earthobservatory.nasa.gov/ContentWOC/images/globaltemp/global_gis_1945-1949.png'\n",
+    "gis_2015 = 'https://earthobservatory.nasa.gov/ContentWOC/images/globaltemp/global_gis_2015-2019.png'\n",
+    "\n",
+    "pn.Fade(gis_1945, gis_2015)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "16d0db31",
+   "metadata": {},
+   "source": [
+    "It can also support more than two objects to fade between."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c1a62e95",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gis_1945 = 'https://earthobservatory.nasa.gov/ContentWOC/images/globaltemp/global_gis_1945-1949.png'\n",
+    "gis_1980 = 'https://earthobservatory.nasa.gov/ContentWOC/images/globaltemp/global_gis_1980-1984.png'\n",
+    "gis_2015 = 'https://earthobservatory.nasa.gov/ContentWOC/images/globaltemp/global_gis_2015-2019.png'\n",
+    "\n",
+    "pn.Fade(gis_1945, gis_1980, gis_2015)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9af72676-c50d-4c61-820d-048daf8872e9",
+   "metadata": {},
+   "source": [
+    "The layout can compare any type of component, e.g. here we will compare two violin plots:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c6187cb3-21a1-4900-be93-02ce3e480ff4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas\n",
+    "\n",
+    "penguins = pd.read_csv('https://datasets.holoviz.org/penguins/v1/penguins.csv')\n",
+    "\n",
+    "pn.Fade(\n",
+    "    penguins[penguins.species=='Chinstrap'].hvplot.violin('bill_length_mm', color='#00cde1'),\n",
```

Comment:
The violin plot requires scipy to be installed. It might be an idea to use a plot type that does not require scipy installed as

- Some users might get a `ImportError: univariate_kde operation requires SciPy to be installed.`
- It would make the notebook much faster to load on  Panelite.